pkgs/development/php-packages/composer/default.nix

```diff
@@ -1,13 +1,11 @@
 { mkDerivation, fetchurl, makeWrapper, unzip, lib, php }:
-let
+
+mkDerivation rec {
   pname = "composer";
   version = "2.2.7";
-in
-mkDerivation {
-  inherit pname version;
 
   src = fetchurl {
-    url = "https://getcomposer.org/download/${version}/composer.phar";
+    url = "https://github.com/composer/composer/releases/download/${version}/composer.phar";
```

Comment:
This isn't needed. The bot already updates composer. We should fetch from upstream's preferred source unless there are issues with it.

Reply:
Indeed, this might not be mandatory, but wouldn't it be better to get everything from Github ?